frontend/src/v5/services/intl.ts

```diff
@@ -74,6 +74,6 @@ export const getIntlProviderProps = () => ({
 			return;
 		}
 
-		console.error(error);
+//		console.error(error);
```

Comment:
👀

Reply:
I have a repressed desire to remove that line (?)